src/services/meta.py

```
@@ -0,0 +1,256 @@
+                                "feed_metadata_url": Settings.METADATA_URL,
+                            },
+                        )
+
+                        data = BytesIO(await response.read())  # Zip file
+                        with zipfile.ZipFile(data, "r") as zf:
+
+                            with sentry_sdk.start_span(
+                                name="Load agencies",
+                                op="load.agencies",
+                            ):
+                                with logfire.span("Load agencies"):
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)